proposals/0389-dynamic-min-deposit.md

```diff
@@ -0,0 +1,249 @@
+---
+simd: '0389'
+title: Reduce account creation constant
+authors:
+  - Igor Durovic (anza)
+category: Standard
+type: Core
+status: Idea
+created: 2025-10-27
+feature: (fill in with feature key and github tracking issues once accepted)
+supersedes:
+superseded-by:
+extends:
+---
+
+## Summary
+
+Reduce the minimum deposit rate by 10x (set baseline
+`min_deposit = 0.1 * min_balance_legacy`) and make this the new floor.
```

Comment:
Since we're now likely going to have a constant reduction before the controller, we may want to specify either an exact number for the new min, or specify the exact value for the `min_balance_legacy`. As I believe we are not intending to do a constant reduction, then an additional 10x reduction. This applies to the whole PR.